inscriptions/ISic000389.xml

```diff
@@ -185,20 +190,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141198" xml:lang="la">
+            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141198" xml:lang="la" resp="#JH">
                 <ab>
-                    <lb n="1"/>
-                    MN<lb n="2"/>AR
-                    <lb n="3"/>
-                    <hi rend="ligature">V̂B</hi>
+                    <lb n="1"/>MN
+                    <lb n="2"/>AR
+                    <lb n="3"/><hi rend="ligature">V̂B</hi>
```

Comment:
because you have encoded ligature there is no need to include the leiden symbol for ligature also. However, what you also want to try and capture is the rotation of the letters, and for this you'd put a style on the line direction:
`<lb n="3" style="line-direction:downwards"/>`